src/arch/riscv/RiscvISA.py

```diff
@@ -107,7 +108,11 @@ class RiscvISA(BaseISA):
         ELEN in Ch. 2 of RISC-V vector spec",
     )
     privilege_mode_set = Param.PrivilegeModeSet(
-        "MSU",
+        "MSU",  # set MHSU to enable hypervisor (H-extension)
+        # All timing CPUs are NOT supported in MHSU currently
```

Comment:
```suggestion
        # No timing CPUs are supported in MHSU currently
        # *ONLY THE ATOMIC CPU / ATOMIC MEMORY* is supported
```

Let me know if that is a misinterpretation. The comment needs to be a bit more explicit than it is now.